pkgs/tools/networking/ssl-proxy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildGo118Module, fetchFromGitHub }:
+buildGo118Module rec {
+  version = "0.2.7";
+  pname = "ssl-proxy";
+
+  src = fetchFromGitHub {
+    owner = "suyashkumar";
+    repo = "ssl-proxy";
+    rev = "v${version}";
+    hash = "sha256-c9BLdDlkrg1z1QrO+vEAVyPtrV/nQcYlGXFmwfAOSpQ=";
+  };
+
+  vendorHash = "sha256-310K9ZSxy/OQ4HYFCcHQaj4NQwzATrOZ2YkhiSkhY5I=";
+
+  doCheckPhase = true;
```

Comment:
I think you mean `doCheck`. But that should be enabled for go modules by default.

Reply:
Oh yeah, my bad. Programming by coincidence :pensive:.  

I found the line in the `buildGoModule` implementation: `doCheck = args.doCheck or true;` 

Going back to my code though, should I set `doCheck = true;` to be more explicit?

Reply:
I dont see other go package do it, so I would say no.